SPECS/sgx-backwards-compatability/sgx-backwards-compatability.spec

```diff
@@ -0,0 +1,46 @@
+Summary:        Custom udev rules to transition SGX applications from the DCAP driver to in-kernel SGX support
+Name:           sgx-backwards-compatability
+Version:        1.0.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+Source0:        99-sgx-backwards-compatability.rules
+Source1:        LICENSE
+
+Requires(pre):  %{_sbindir}/groupadd
+Requires(post): systemd-udev
+
+%description
+A main-line kernel with SGX support will expose dev nodes to flat files.
```

Comment:
Nit - wording for the package description reads a little awkwardly to me. Can we highlight what this package is supplying more prominently?